types/truffle-typings/index.d.ts

```diff
@@ -2,10 +2,14 @@
  * Include generated truffle typings for tests.
  */
 
-/// <reference types="../../build/truffle-typings/merge" />
+/// <reference types="../../build/truffle-typings/types" />
 
 export * from "../../build/truffle-typings";
 
-export interface Linkable<T> extends Truffle.Contract<T> {
-  link(lib: string, address: string): Promise<void>;
+declare global {
+  namespace Truffle {
+    interface Artifacts {
+      require(name: string): Truffle.Contract<unknown>;
+    }
+  }
 }
```

Comment:
I guess this is going to conflict with my deposit migration #802. Should be easy enough to deal with the conflict on either side.

Reply:
Trivial to merge.